assets/js/hooks/useKeyCodesInside.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * `useKeyCodesInside` hook.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useKey } from 'react-use';
+
+export const useKeyCodesInside = ( keyCodes, ref, handler ) => {
+	useKey(
+		( event ) => keyCodes.includes( event.keyCode ) &&
+		ref.current.contains( event.target ),
```

Comment:
Let's not use implicit return here, because it makes this a lot tougher to read. How about:

```js
( event ) => {
  return keyCodes.includes( event.keyCode ) && ref.current.contains( event.target );
}, handler
```

The implicit return over multiple lines is too vague to my eyes.